cloudinit/config/cc_phone_home.py

```diff
@@ -88,6 +88,7 @@
             """
         ),
     ],
+    "activate_by_schema_keys": [],
```

Comment:
```suggestion
    "activate_by_schema_keys": ["phone_home"],
```

Reply:
Fixed. The reason to not added in the first place was that this module makes use of the `args` argument, and I wasn't sure which entry-point passes `args` through.

Reply:
+1 @aciba90 the only way to pass through args is via `cloud-init single -c cc_phone_home <some_yaml_file>` which avoids your skip logic over in cloudinit.config.modules anyway. I too had to spend time digging that path up yesterday too.

Reply:
not a common use case for sure